docs/lib/storage/fragments/flutter/getting-started/10_preReq.md

```diff
@@ -6,7 +6,7 @@
 
 <amplify-callout warning>
 
-If you are using Flutter (2.2.0), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
+If you are using Flutter (2.2.0 or greater), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
```

Comment:
Not a blocker by any means but it may be worth extracting this callout into a fragment so that if we need to update it again in the future, we don't have to update it in 10 different places.